README.md

```diff
@@ -22,7 +22,7 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Run Pyre Action
-        uses: facebook/pyre-action
+        uses: facebook/pyre-action@v0.0.1
```

Comment:
Would you be able to change it to something like this:
```
      - name: Run Pyre Action
        # update with your version (e.g. `facebook/pyre-action@v0.0.1`)
        uses: facebook/pyre-action
```

Reply:
I can see why you want to avoid hard coding the version number, and adding a comment makes it pretty clear that it need an adjustment so the error doesn’t come out of nowhere.

I slightly adjusted the message: `# update with the desired version of the action (e.g. 'facebook/pyre-action@v0.0.1')`.